src/modules/internet/index.ts

```diff
@@ -804,4 +806,111 @@ export class InternetModule extends ModuleBase {
       this.faker.definitions.internet.emoji[emojiType]
     );
   }
+
+  /**
+   * Generates a random JWT (JSON Web Token).
+   *
+   * @param options Options object.
+   * @param options.header Header part of the token.
+   * @param options.header.alg Algorithm header parameter.
+   * @param options.payload Payload part of the token.
+   * @param options.payload.iss Issuer claim.
+   * @param options.payload.sub Subject claim.
+   * @param options.payload.aud Audience claim.
+   * @param options.payload.jti JWT ID claim.
+   * @param options.refDate The date to use as reference point for the newly generated date.
+   *
+   * @see {@link https://datatracker.ietf.org/doc/html/rfc7519}
+   *
+   * @example
+   * faker.internet.jwt()
+   * faker.internet.jwt({ header: { alg: 'HS256' }})
+   * faker.internet.jwt({ payload: { iss: 'Acme' }})
+   * faker.internet.jwt({ payload: { iss: 'Acme', sub: 'Subject' }})
+   * faker.internet.jwt({ payload: { iss: 'Acme', sub: 'Subject', aud: 'Audience' }})
+   * faker.internet.jwt({ payload: { iss: 'Acme', sub: 'Subject', aud: 'Audience', jti: 'JWT ID' }})
+   *
+   * @since 8.0.0
+   */
+  jwt(options?: {
+    header?: {
+      /**
+       * Algorithms to use.
+       *
+       * @default Object.values(faker.definitions.internet.jwt.alg)
+       */
+      alg: string;
+    };
+    /**
+     * Payload part of the token.
+     */
+    payload?: {
+      /**
+       * Issuer claim.
+       *
+       * @default faker.company.name()
+       */
+      iss?: string;
+      /**
+       * Subject claim
+       *
+       * @default faker.string.uuid()
+       */
+      sub?: string;
+      /**
+       * Audience claim
+       *
+       * @default faker.string.uuid()
+       */
+      aud?: string;
+      /**
+       * JWT ID claim
+       *
+       * @default faker.string.uuid()
+       */
+      jti?: string;
+    };
+    /**
+     * The date to use as reference point for the newly generated date.
+     *
+     * @default faker.defaultRefDate()
+     */
+    refDate?: string | Date | number;
+  }): string {
```

Comment:
We usually default the options object to an empty object in the method parameters.

```suggestion
  } = {}): string {
```